CMakeLists.txt

```diff
@@ -218,6 +224,7 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
 	add_feature_info(Graphviz  	WITH_GRAPHVIZ  		"Graphviz Graphs")
 	add_feature_info(Sundials  	WITH_SUNDIALS  		"Sundials solvers")
 	add_feature_info(PYBIND 	WITH_PYBIND 		"Use DPsim as a PYBIND module")
+	add_feature_info(KLU		WITH_KLU			"A partially refactoring LU solver")
```

Comment:
Description rather something like "Use customized KLU module"? KLU might be of interest to the user even without the partial refactorisation capabilities that are provided by our customized KLU module.

Reply:
Done